app/metrics.yaml

```diff
@@ -16,10 +16,72 @@ distribution:
       - baseline
       - events
       - metrics
+      - session_end
     bugs:
       - https://github.com/MozillaReality/FirefoxReality/issues/1420
     data_reviews:
       - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
     notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+url:
+  domains:
+    type: counter
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many domains are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  visits:
+    type: counter
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many URL links are visited in a session.
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+  query_type:
+    type: labeled_counter
+    send_in_pings:
+      - session_end
+    description: >
+      Counting how many URLs are visited in a session, by input method.
+    labels:
+      - type_link
+      - type_query
+      - voice_query
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2230
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/1854#issuecomment-546214568
+    notification_emails:
+      - fxr-telemetry@mozilla.com
+      - dmu@mozilla.com
+    expires: "2020-05-01"
+searches:
+  counts:
```

Comment:
This is lacking the `send_in_pings` and it's not being sent in your new ping.

Reply:
I was thinking the searches.count can be sent in metrics. We are interested in collecting it in daily instead of by sessions, and it will go by default into the metrics ping. Therefore, I ignored `send_in_pings`, does it make sense?

Reply:
It does, I wanted to make sure that was your intent ;) Since that's the intent, it's fine!